### DISCUSSION TITLE
button/link with target=_blank?

### DISCUSSION THREAD

**Discussion by Ra-Lo** [orchidsoftware/platform#2256] [Q&A]
Hello,
Is there any way to open the link of the the button/link element in a new window?
In Orchid backend I have an input page to edit a record. There I want to have a preview link to see the record in the frontend.
I would like to open this link in a new window.

Thanks and greetings

**Comment by tabuna** [comment]
You can use the link element to open the page in a new tab:

```php
use Orchid\Screen\Action\Link;

Link::make('Open Docs')
    ->href('https://orchid.software/en/docs/')
    ->target('_blank')
```

**Reply by Ra-Lo** [reply]
many thanks :-)
